### PR TITLE
Parser cleaning

### PR DESCRIPTION
**Pull request by dixneuf19**
Refactor Parser into several "parselet" (StatementParser)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ apply plugin: 'application'
 apply plugin: 'java'
 apply plugin: 'org.springframework.boot'
 apply plugin: 'io.spring.dependency-management'
-apply from: rootProject.file("gradle/install-git-hooks.gradle")
```
Comment (remicalixte) [inline build.gradle]: Is this on purpose

Comment (dixneuf19) [reply build.gradle]: Yes see https://github.com/jua-cs/jua/pull/8/commits/c5ca41558bcf6d51230ea530f52bc8efe2367648 for more details

Comment (remicalixte) [reply build.gradle]: great job!

Comment (remicalixte) [reply build.gradle]: most important change of the PR cc @sfluor

Comment (sfluor) [reply build.gradle]: I think we can leave it it's not automatic you have a o explicitly run installGitHooks to install them

Comment (dixneuf19) [reply build.gradle]: No, each `gradle build` makes it reappears ! :angry:

**Pull request merged**